scipy/linalg/_decomp_cholesky.py

```diff
@@ -6,10 +6,16 @@
 from ._misc import LinAlgError, _datacopied
 from .lapack import get_lapack_funcs
 
+from scipy._lib._array_api import array_namespace, is_numpy
+
 __all__ = ['cholesky', 'cho_factor', 'cho_solve', 'cholesky_banded',
            'cho_solve_banded']
 
 
+def arg_err_msg(param):
```

Comment:
will be renamed and imported from `_array_api.py` in a follow up to https://github.com/scipy/scipy/pull/19005.